RELEASE_NOTES.md

```diff
@@ -1,5 +1,16 @@
 ﻿RELEASE NOTES
 
+**Version**: 1.1
+
+This versions requires:
+* Shotgun for Unity 1.1 (com.unity.integrations.shotgun)
+
+NEW FEATURES
+* Direct Unity project launch(no need for Unity Hub)
+* Now working in-process
```

Comment:
I'm not sure if we need to mention this one here. Or mention that "we updated the API to the 4.0.0 version of the Python for Unity package"